edx/edx-enterprise
# ENT-1463 Course duration in view from data consent screen

**Description:** This PR adds course duration to the view from data consent screen.

**JIRA:** https://openedx.atlassian.net/browse/ENT-1463

Test Instructions:

1: Enroll in a course e.g. http://edx.devstack.lms:18000/enterprise/1a846f05-42bf-4cfe-91bf-30bbb937ffbb/course/course-v1:edX+DemoX+2019-T1/enroll/
2: Click Course Details in 'Confirm Your Course' page
3: The new field should appear as:
<img width="1057" alt="course-duration" src="https://user-images.githubusercontent.com/34648393/57841906-73eb1a80-7799-11e9-92db-decb36d9b047.png">
